config/build/webpack.config.common.js

```diff
@@ -34,7 +35,7 @@ export const createResolveConfigLoader = () => ({
 
 export function config(options) {
     return {
-        devtool: options.dev && 'source-map',
+        devtool: options.dev && 'eval-cheap-module-source-map',
```

Comment:
it seems @christophehurpeau _"pèse dans le game"_ 

![](https://d3vv6lp55qjaqc.cloudfront.net/items/3T401r3j3r3W3x2y0G10/Image%202017-04-04%20at%203.17.55%20PM.png?X-CloudApp-Visitor-Id=35667)